include/boost/math/policies/policy.hpp

```diff
@@ -837,54 +834,54 @@ struct precision
 template <class Policy>
 struct precision<BOOST_MATH_FLOAT128_TYPE, Policy>
 {
-   typedef boost::integral_constant<int, 113> type;
+   typedef std::integral_constant<int, 113> type;
 };
 
 #endif
 
 namespace detail{
 
 template <class T, class Policy>
-inline BOOST_MATH_CONSTEXPR int digits_imp(boost::true_type const&) BOOST_NOEXCEPT
+inline constexpr int digits_imp(std::true_type const&) noexcept
 {
 #ifndef BOOST_NO_LIMITS_COMPILE_TIME_CONSTANTS
```

Comment:
Just a heads up that BOOST_NO_LIMITS_COMPILE_TIME_CONSTANTS can be assumed to NOT be set post C++11, so these pp-branches can be chopped down.